Local/Factoids.py

```diff
@@ -74,6 +74,8 @@ def handle(self, message):
             txt = "^('-')^ ^('-')^ v('-')v v('-')v <('-'<) (>'-')> <('-'<) (>'-')>"
         elif command == '!slowclap':
             txt = 'https://p.bastelfreak.de/yVB9/'
+        elif command == '!eyeballs':
+            txt = 'http://i2.kym-cdn.com/photos/images/original/000/282/317/50d.png'
```

Comment:
@bastelfreak Is linking to random image URLs ok, or do you prefer to host them yourself?